pkg/test/framework/components/istio/istio.go

```diff
@@ -53,12 +65,7 @@ func SetupOnKube(i *Instance, cfg *Config) framework.SetupFn {
 // Deploy deploys (or attaches to) an Istio deployment and returns a handle. If cfg is nil, then DefaultConfig is used.
 func Deploy(ctx resource.Context, cfg *Config) (Instance, error) {
 	if cfg == nil {
-		c, err := DefaultConfig(ctx)
-		if err != nil {
-			return nil, err
-		}
-
-		cfg = &c
+		return nil, fmt.Errorf("deploy Istio receives an empty config")
```

Comment:
I'd still keep the default config here. The deploy command can be called from regular tests directly as well.

Reply:
sure. also make it more resilient.